.github/workflows/release-please.yml

```diff
@@ -54,6 +54,9 @@ jobs:
     needs: release-please
     runs-on: ubuntu-latest
     if: ${{ needs.release-please.outputs.release_created }}
+    permissions:
```

Comment:
I believe these are the only permissions we'll need, but I didn't have an easy way to confirm.